tests/contrib/test_multistore_file.py

```diff
@@ -186,9 +187,8 @@ def test_multistore_no_symbolic_link_files(self):
             'user-agent/1.0',
             ['some-scope', 'some-other-scope'])
         try:
-            self.assertRaises(
-                locked_file.CredentialsFileSymbolicLinkError,
-                store.get)
+            with self.assertRaises(CredentialsFileSymbolicLinkError):
```

Comment:
Fixed.  There wasn't a similar example in the Google Style Guide, so I went with the second acceptable example in the [PEP8 guidelines](https://www.python.org/dev/peps/pep-0008/#indentation).